### DISCUSSION TITLE
Don't update the modification date when importing a backup

### DISCUSSION THREAD

**Discussion by edas** [standardnotes/forum#2598] [Suggestions]
I tried to migrate my data from another software, and set the previous original last modification date. I see briefly the correct modification dates in the client UI. The modification dates are however reseted once it synchronizes with the server.

This behaviour make nearly impossible to import an large archive (everything will be reseted, so unusable).

---- 

I am using https://github.com/edas/simplenote2standardnote

Example of an backup file I am trying to import:

```json
{
  "items": [
    {
      "uuid": "6fe8b8c0-b543-11e8-9721-7725ee042416",
      "created_at": "2018-09-10T21:48:37.611Z",
      "updated_at": "2019-06-14T10:05:36.406Z",
      "content_type": "Note",
      "content": {
        "title": "my title",
        "text": "my content",
        "references": [

        ],
        "trashed": false,
        "preview_plain": "my preview…",
        "preview_html": null,
        "appData": {
          "org.standardnotes.sn": {
            "client_updated_at": "2019-06-14T10:05:36.406Z"
          }
        }
      }
    }
  ]
}
```

**Comment by atmoio** [comment]
`updated_at` is reserved by our app and syncing engine, and cannot be changed other than by the app itself. This is why we added `client_updated_at`. This is a user configureable property, and where you should store the modification date you want displayed in the UI.

**Comment by edas** [comment]
Understood, thank you for the answer. However I did also set the `client_updated_at` in my example, but still have all notes with a new last modification date in the UI.

**Comment by atmoio** [comment]
Actually it does seem like we modify the client_updated_at upon import: 

https://github.com/standardfile/sfjs/blob/master/lib/app/lib/modelManager.js#L633
https://github.com/standardfile/sfjs/blob/master/lib/app/lib/modelManager.js#L340

I've made an issue here: https://github.com/standardnotes/web/issues/303
